backend/README.md

```diff
@@ -12,3 +12,35 @@ Python service for the InferGPT backend.
 - To install all project dependencies (listed in `requirements.txt`) run `pip install -r requirements.txt`.
 - Once all dependencies have been pulled run `uvicorn api:app --port 8250` to start the app. 
 - Check the backend app is running at [http://127.0.0.1:8250/health](http://127.0.0.1:8250/health).
+
+## Backend Linting
+
+Ruff is being used for the backend linting. See the docs: [here](https://docs.astral.sh/ruff/)
+
+Make sure the ruff dependency is downloaded, it should have been included in `requirements.txt`.
+
+Run the following command to check:
+
+```bash
+ruff --version
+```
+
+If it is not installed use the follwoing command:
+
+```bash
+pip install ruff
+```
+
+If ruff is installed correctly, the python files in the backend will be checked using the following command
+
+```bash
+ruff check backend
```

Comment:
is `backend` here needed if you are already in the backend directory?

Reply:
No, should I assume that the person is in the backend directory then?

Reply:
I think from the backend readme, yes :slightly_smiling_face: but can always add before the code block from this directory run:   sort of thing